netket/optimizer/preconditioner.py

```diff
@@ -64,9 +67,11 @@ def __init__(self, lhs_constructor, solver, *, solver_restart=False):
         self.solver = solver
         self.solver_restart = solver_restart
 
-    def __call__(self, vstate: VariationalState, gradient: PyTree) -> PyTree:
+    def __call__(
+        self, vstate: VariationalState, gradient: PyTree, step_value: Number
```

Comment:
```suggestion
        self, vstate: VariationalState, gradient: PyTree, step_value: Number = 0
```
It would improve backwards compatibility if this call doesn't require a `step_value`

Reply:
a better default value would be `None`, so that it will throw some error if someone tries to access it when it's needed.

Reply:
sounds good